atests/http_server/helpers.py

```diff
@@ -110,15 +110,16 @@ def get_files():
 
     files = dict()
 
-    for k, v in request.files.items():
-        content_type = request.files[k].content_type or 'application/octet-stream'
-        val = json_safe(v.read(), content_type)
-        if files.get(k):
-            if not isinstance(files[k], list):
-                files[k] = [files[k]]
-            files[k].append(val)
-        else:
-            files[k] = val
+    for k in request.files.keys():
```

Comment:
you also had to fix httpbin?

Reply:
Yes, as this only returned one item the way it was written. According to what I found out, you need to use the **getlist(key)** method to get all files for one key.